### PR TITLE
Feature 164 fix texts in admin mails (#244)

### PR DESCRIPTION
**Pull request by sven-dmlr**
Improved email texts and fixed some typos:

- tweaked email texts and subjects
- added environment url where possible
- spell corrections
- tested+verified locally
- builds are green
---
<sup>Sven Dolderer <sven.dolderer@daimler.com>, Daimler TSS GmbH, [imprint](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)</sup>

### CODE REVIEW

File: sechub-notification/src/main/java/com/daimler/sechub/domain/notification/superadmin/InformAdminsThatSchedulerJobProcessingHasBeenDisabledService.java [hunk 1]
```diff
@@ -38,8 +38,8 @@ public void notify(String baseUrl) {
 
 	private String createEmailContent(String baseUrl) {
 		StringBuilder emailContent = new StringBuilder();
-		emailContent.append("Scheduler job processing has been disabled at sechub for environment (base url):").append(baseUrl).append("\n");
-		emailContent.append("WARNING: Users can still add new Jobs, but will wait for execution until scheduler job processing will be enabled again!");
+		emailContent.append("Scheduler job processing has been disabled at SecHub for environment (base url): " + baseUrl + "\n\n");
```
Comment (de-jcup) [inline sechub-notification/src/main/java/com/daimler/sechub/domain/notification/superadmin/InformAdminsThatSchedulerJobProcessingHasBeenDisabledService.java:41]: Please look at https://www.baeldung.com/java-strings-concatenation
where mixing up append and + operatior is mentioned as bad practice - as it is...

File: sechub-integrationtest/src/main/java/com/daimler/sechub/integrationtest/internal/MockEmailAccess.java [hunk 2]
```diff
@@ -86,7 +101,7 @@ public MockEmailEntry findMailOrFail(String email, String subject, int maxSecond
 			sb.append("Did not found mail containing:\n-emailadress (TO/CC/BCC): ");
 			sb.append(email);
 			sb.append("\n-subject: '");
-			sb.append(subject);
+			sb.append(subjectRegexp);
```
Comment (de-jcup) [inline sechub-integrationtest/src/main/java/com/daimler/sechub/integrationtest/internal/MockEmailAccess.java:104]: Hmm.. we use this variable for regular expressions but also for normal string search - when reading this I was thinking the parameter would be only for regular expression. Maybe we should still use "subject" or "subjectContent" instead?

File: sechub-notification/src/main/java/com/daimler/sechub/domain/notification/owner/InformOwnerThatProjectHasBeenDeletedNotificationService.java [hunk 3]
```diff
@@ -27,20 +27,20 @@
...
 		}
-		StringBuilder emailContent = new StringBuilder();
-		emailContent.append("Project ").append(projectMessage.getProjectId()).append(" has been deleted.\n");
-		emailContent.append("This means that all report data has been deleted, and no longer access for sechub scans for this project is possible.\n\n");
-		emailContent.append("The project owner has been already informed.\n");
+		SimpleMailMessage message = factory.createMessage("A SecHub project you were the owner was deleted: " + projectMessage.getProjectId());
 
-		SimpleMailMessage message = factory.createMessage("A SecHub project you were the owner was deleted");
+		StringBuilder emailContent = new StringBuilder();
+		emailContent.append("Your project '" + projectMessage.getProjectId() + "' in environment " + baseUrl + "\n");
```
Comment (de-jcup) [inline sechub-notification/src/main/java/com/daimler/sechub/domain/notification/owner/InformOwnerThatProjectHasBeenDeletedNotificationService.java:41]: Please use append
see https://www.baeldung.com/java-strings-concatenation

File: sechub-notification/src/main/java/com/daimler/sechub/domain/notification/user/UserDeletedNotificationService.java [hunk 4]
```diff
@@ -27,7 +27,9 @@
 	
 	public void notify(UserMessage userMessage) {
 		StringBuilder emailContent = new StringBuilder();
-		emailContent.append("Your account has been removed by an administrator.\n");
+		emailContent.append("Your account '" + userMessage.getUserId() + "'");
```
Comment (de-jcup) [inline sechub-notification/src/main/java/com/daimler/sechub/domain/notification/user/UserDeletedNotificationService.java:30]: Please use append
https://www.baeldung.com/java-strings-concatenation

File: sechub-notification/src/main/java/com/daimler/sechub/domain/notification/superadmin/InformAdminsThatSchedulerJobProcessingHasBeenDisabledService.java (hunk 1 above)
Comment (sven-dmlr) [reply sechub-notification/src/main/java/com/daimler/sechub/domain/notification/superadmin/InformAdminsThatSchedulerJobProcessingHasBeenDisabledService.java:41]: changed all "+" into .append()

File: sechub-notification/src/main/java/com/daimler/sechub/domain/notification/owner/InformOwnerThatProjectHasBeenDeletedNotificationService.java (hunk 3 above)
Comment (sven-dmlr) [reply sechub-notification/src/main/java/com/daimler/sechub/domain/notification/owner/InformOwnerThatProjectHasBeenDeletedNotificationService.java:41]: changed all "+" into .append()

File: sechub-notification/src/main/java/com/daimler/sechub/domain/notification/user/UserDeletedNotificationService.java (hunk 4 above)
Comment (sven-dmlr) [reply sechub-notification/src/main/java/com/daimler/sechub/domain/notification/user/UserDeletedNotificationService.java:30]: changed all "+" into .append()

File: sechub-integrationtest/src/main/java/com/daimler/sechub/integrationtest/internal/MockEmailAccess.java (hunk 2 above)
Comment (sven-dmlr) [reply sechub-integrationtest/src/main/java/com/daimler/sechub/integrationtest/internal/MockEmailAccess.java:104]: I this case, it is really the regexp used for searching the subject.
Will improve the message \n-subject:... so it's more clear.

**Comment by de-jcup** [comment]
Review has been done, PR will be merged
---
<sup>Albert Tregnaghi <albert.tregnaghi@daimler.com>, Daimler TSS GmbH, [imprint](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)</sup>

**Pull request merged**